ci/auto-pr/conv_proj_version_to_branch

```diff
@@ -12,8 +12,7 @@ versions.inject(Set.new) {|acc, v|
       # e.g. project: "ScalarDB 4.0.0" -> branch: "master"
       #
       # This GitHub project corresponds to `main`/`master` branch.
-      # The target change is already merged to the default branch.
-      # Nonthing to do.
+      acc << "master"
```

Comment:
Sorry, I didn't get why this change is needed. Could you tell me what would happen without the change?

Reply:
This change actually doesn't affect the behavior. With the change in this PR, we remove the base branch of the original PR from the target branch list. So we no longer need the special care for the `main`/`master` branch in this `conv_proj_version_to_branch` script. What do you think?

Reply:
I see. My concern is this default branch name is different among ScalarDB projects, and ideally it should be passed by `auto-pr.yaml` that gets the default branch name from `github.event.repository.default_branch`. But it takes some effort and introduces a bit of complexity. So it would be better to make it simple by removing this.

Reply:
> My concern is this default branch name is different among ScalarDB projects

I thought the same thing, and for now, it seems we'll need to use slightly modified scripts (with different default branch names) in other repositories. In the future, we can unify the default branch name across the repositories. 

Moreover, considering that we create PRs from not just the default branches (`master` or `main`) but also from support/release branches (`3` and `3.x`), the following comment may not always apply:
```bash
      # The target change is already merged to the default branch.
      # Nonthing to do.
```

Given this, I feel the special handling to the `master`/`main` branches strange here. That's why I made the change.

What do you think?

Reply:
Sounds reasonable to me 👍